templates/show-worklist.html

```diff
@@ -5,6 +5,7 @@
     <script src="https://tools-static.wmflabs.org/cdnjs/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.js"></script>
     <link rel="stylesheet" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/awesomplete/1.1.2/awesomplete.css" />
     <script src="https://tools-static.wmflabs.org/cdnjs/ajax/libs/awesomplete/1.1.2/awesomplete.js" async></script>
+    <script src="{% static 'js/arrive.js' %}"></script>
```

Comment:
https://tools-static.wmflabs.org/cdnjs/ajax/libs/arrive/2.4.1/arrive.js, here and in show-task.html please.

Reply:
Done!